deploy-olm-operator.yml

```diff
@@ -4,17 +4,16 @@
   become: false
   gather_facts: false
 
-  vars_files:
-    - "{{ playbook_dir }}/../../../config/ansible/vault.yml"
 
   vars:
+    vault_file_path: "{{ vault_file_path | default(playbook_dir +'/../../../config/ansible/vault.yml') }}"
     run_upstream: false
     openshift_namespace: "test-operator"
     operator_work_dir: "/home/jenkins/agent/test-operator"
     scorecard_cr_dir: "/home/jenkins/agent/scorecard-cr-files"
     work_dir: "{{ lookup('env', 'WORKSPACE') }}"
     olm_operator_files_path: "{{ work_dir }}/olm-operator-files"
-    quay_token: "{{ cvp_vault.quay_token }}"
+    quay_token: "{{ vault_file_path.quay_token }}"
```

Comment:
@jsztuka if we need to load wjole vaout file jsut for `quay_token` than it seems to me like overkill.
We can pass quay token with `ansible-playbook --extra-vars`

Reply:
--extra-vars is more appealing for me, we don't use any other secret besides this one in the playbook.